neusomatic/python/filter_candidates.py

```diff
@@ -56,38 +82,85 @@ def filter_candidates(candidate_record):
 
                 if loc not in records:
                     records[loc] = []
-                if ref == "N" or "\t".join(line.split()[0:5]) \
-                        not in map(lambda x: "\t".join(x[-1].split()[0:5]), records[loc]):
-                    records[loc].append([chrom, pos, ref, alt, dp, ro, ao, mq_, bq_, st_, ls_, rs_,
-                                         nm_, as_, xs_, pr_, cl_, line])
-                elif "\t".join(line.split()[0:5]) \
-                        in map(lambda x: "\t".join(x[-1].split()[0:5]), records[loc]):
+                if ref == "N" or "\t".join(line.split()[0:5]) not in map(
+                    lambda x: "\t".join(x[-1].split()[0:5]), records[loc]
+                ):
+                    records[loc].append(
+                        [
+                            chrom,
```

Comment:
Would the formatter have options to control for such formatting? For example, the number of variables to put on one line (here, it seems to be 1)?